Sources/PT.PM.Patterns/JavaPatterns.cs

```diff
@@ -448,8 +448,67 @@ public IEnumerable<PatternRoot> CreateJavaPatterns()
                     },
                     Body = new PatternArbitraryDepth
                     {
-                        Pattern = new PatternMethodDeclaration(
-                            Enumerable.Empty<PatternUst>().ToList(), new PatternIdRegexToken(".+"), false)
+                        Pattern = new PatternOr
+                        {
+                            Patterns = new List<PatternUst>
+                            {
+                                new PatternMethodDeclaration(Enumerable.Empty<PatternUst>(), new PatternIdRegexToken(@"\w+"), false),
+                                new PatternOr
+                                {
+                                    Patterns = new List<PatternUst>
+                                    {
+                                        new PatternMethodDeclaration
+                                        {
+                                            Modifiers = Enumerable.Empty<PatternUst>().ToList(),
```

Comment:
Why not a simple `new List<PatternUst>()`?
Actual for further cases too.

Reply:
I think it's slightly more efficient than `new List<PatternUst>()`.

Reply:
Why? `ToList()` creates a new instance anyway. Moreover, in this case, readability is preferable over performance.

Reply:
Ok